intranet/apps/eighth/views/admin/groups.py

```diff
@@ -32,6 +32,10 @@ def add_group_view(request):
     if request.method == "POST":
         form = QuickGroupForm(request.POST)
         if form.is_valid():
+            if form.cleaned_data["name"].startswith("admin_") and not request.user.is_superuser:
```

Comment:
Could you wrap these ifs into a separate helper functions like `admin_group_permission`?

Reply:
I'm not sure what you're asking. Is adding a `User.has_admin_group_permission` field instead of hardcoding `is_superuser` good enough?

Reply:
I am thinking of a function of the form:
`admin_group_permission(group, request.user)`

If the group starts with `admin_` and the user is a superuser, it returns true.

Reply:
What about `User.can_manage_group(group: Union[Group, str]) -> bool`?

Reply:
That is also acceptable.